src/lib/swapper/tradeExecution.ts

```diff
@@ -32,12 +32,15 @@ export class TradeExecution {
       tradeQuote,
       stepIndex,
       slippageTolerancePercentageDecimal,
-    }: CommonTradeExecutionInput,
+      ...rest
+    }: CommonTradeExecutionInput | EvmTransactionExecutionInput,
     buildSignBroadcast: (
       swapper: Swapper & SwapperApi,
       args: CommonGetUnsignedTransactionArgs,
     ) => Promise<string>,
   ) {
+    const supportsEIP1559 = 'supportsEIP1559' in rest ? rest.supportsEIP1559 : undefined
```

Comment:
not a fan of `in` for low-level performance reasons (irrelevant here), but also this (is relevant):
![image](https://github.com/shapeshift/web/assets/125113430/d6add5ba-d4c0-47e1-8de1-edebfe298e05)

Reply:
I believe this one is fine, since `supportsEIP1559` is a boolean, it being undefined would evaluate to falsy, which is intended @woodenfurniture